api/spec/requests/spree/api/v2/storefront/cart_spec.rb

```diff
@@ -100,4 +100,44 @@
       end
     end
   end
+
+  describe 'cart#empty' do
+    context 'without existing order' do
+      it 'returns status code 404' do
+        headers = { 'Authorization' => "Bearer #{token.token}" }
+        post '/api/v2/storefront/cart/empty', headers: headers
+
+        expect(response.status).to eq(404)
+      end
+    end
+
+    context 'with existing order and line item' do
+      let!(:order) { create(:order, user: user) }
+      let!(:line_item) { create(:line_item, order: order) }
+
+      it 'empties the order' do
+        headers = { 'Authorization' => "Bearer #{token.token}" }
+        post '/api/v2/storefront/cart/empty', headers: headers
+
+        expect(response.status).to eq(200)
+
+        expect(order.line_items.count).to eq(0)
+      end
+    end
+
+    context 'with existing guest order and line item' do
+      let(:guest_token) { 'guest_token' }
+      let!(:order) { create(:order, guest_token: guest_token) }
+      let!(:line_item) { create(:line_item, order: order) }
+
+      it 'empites the order' do
+        headers = { 'Authorization' => "Bearer #{token.token}" }
+        post '/api/v2/storefront/cart/empty', params: { order_token: guest_token }, headers: headers
```

Comment:
@WitekL let's not use `params: { order_token: guest_token }` - it should be passed as `X-Spree-Order-Token` header, also for the guest checkout flow - we shouldn't pass the `Bearer` header at all